### PR TITLE
Private/gokay/debugging improvements

### PR DESCRIPTION
**Pull request by gokaysatir**
* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/js/global.js [hunk 1]
```diff
@@ -1,50 +1,9 @@
...
-			pixels: [0, 0] // This can be larger than the document's size.
-		}
-	},
-	tile: {
-		size: {
-			pixels: [0, 0],
-			twips: [0, 0]
-		}
-	},
+window.app = {
```
Comment (eszkadev) [inline browser/js/global.js:4]: window.app is duplicated here and in docstate.jsm is that intended?

File: browser/src/docstate.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+			pixels: [0, 0] // This can be larger than the document's size.
+		}
+	},
+	tile: {
+		size: {
+			pixels: [0, 0],
+			twips: [0, 0]
+		}
+	},
+	socket: window.app.socket,
```
Comment (eszkadev) [inline browser/src/docstate.js:45]: self reference? window.app.socket = window.app.socket ?

File: browser/js/global.js (hunk 1 above)
Comment (gokaysatir) [reply browser/js/global.js:4]: :) yep, hmm it's not duplicated indeed. Recreated, i needed the same variable. Writing it like that looks much more familiar while reading the code than adding new properties.

Comment (eszkadev) [reply browser/js/global.js:4]: ok, so can't we have only one place with window.app? I fear someone will be adding properties not in the correct place...
or these files are not always present both at the same time?

Comment (gokaysatir) [reply browser/js/global.js:4]: Oh, that's right. We should add some comments there. Unfortunately "global.js" file needs those 2 variables at initialization. So i had to leave those 2 variables there. And the rest of the object is defined in "docstate.js" file.

They will be both present at the same time. I first tried to move "global.js" to outside of html but it's not a good solution for various reasons. The second option was this current solution.

Or we can move those 2 variables outside of "window.app" object.

My take would be adding a comment there to ensure that noone expands the object inside that file.

File: browser/src/docstate.js (hunk 2 above)
Comment (gokaysatir) [reply browser/src/docstate.js:45]: Thanks for the valuable comments :)

File: browser/js/global.js (hunk 1 above)
Comment (eszkadev) [reply browser/js/global.js:4]: yes, good idea, with comment nobody will be confused :)

**Review by eszkadev — APPROVED** [review]

File: browser/src/docstate.js (hunk 2 above)
Comment (merttumer) [reply browser/src/docstate.js:45]: cant we do something like window.app = {...window.app, ...{rest_here}} just to say we are expanding window.app? but of course it does not really matter in terms of functionality.

**Pull request merged**